packages/netdog/netdog.spec

```diff
@@ -0,0 +1,101 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}netdog
+Version: 0.1.0
+Release: 0%{?dist}
+Summary: Bottlerocket network configuration helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source2: netdog-tmpfiles.conf
+
+Source10: run-netdog.mount
+Source11: write-network-status.service
+Source12: generate-network-config.service
+Source13: disable-udp-offload.service
+
+Source20: 00-resolved.conf
```

Comment:
Why start with `Source2`?

Headers in a comment might help enforce the grouping, e.g.:
```
# drop-in configs
...

# units
...

# service configs
...
```